src/components/interpolatedColorLegend.ts

```diff
@@ -109,6 +123,30 @@ export module Component {
       }
     }
 
+    /**
+     * Apply a gradient to the InterpolatedColorLegend.
+     *
+     * @param {boolean} isGradient Whether there should be a gradient or not.
+     *
+     * @returns {InterpolatedColorLegend} The calling InterpolatedColorLegend.
+     */
+     public gradient(isGradient: boolean): InterpolatedColorLegend {
+       this._isGradient = isGradient;
+       return this;
+     }
+
+     /**
+      * Expand the InterpolatedColorLegend to the full length/height of the chart.
+      *
+      * @param {boolean} isExpanded Whether the legend is expanded or not.
+      *
+      * @returns {InterpolatedColorLegend} The calling InterpolatedColorLegend.
+      */
+    public expand(isExpanded: boolean): InterpolatedColorLegend {
+      this._isExpanded = isExpanded;
+      return this;
+    }
```

Comment:
Using the call that I described up above.  However, this does involve 2 pieces of information that is not well described to the user (numSwatches and textHeight).  Maybe the approach above is indeed correct... I'm just afraid of building too little and then having another customer asking for a more configurable call.

That said, if we think that this fixes most of the cases and we're ok with an API break more down the line, we should go with this approach I think